### ISSUE TITLE
Tests fail?

### ISSUE THREAD

**Issue by sorenisanerd** [cweill/gotests#136]
When I run the test suite locally, I get a failure. I see that the test suite runs succesfully in Travis, but I'm not sure what the difference might be. Do you have any ideas?

```soren@potassium:~/src/gotests$ go test ./...
--- FAIL: TestGenerateTests (4.77s)
    gotests_test.go:772: "Receiver is indirect imported struct". GenerateTests(testdata/test037.go) = 
        package testdata
        
        import "testing"
        
        func Test_someIndirectImportedStruct_Foo037(t *testing.T) {
        	tests := []struct {
        		name string
        		smtg *someIndirectImportedStruct
        	}{
        		// TODO: Add test cases.
        	}
        	for _, tt := range tests {
        		tt.smtg.Foo037()
        	}
        }
        , want 
        package testdata
        
        import "testing"
        
        func Test_someIndirectImportedStruct_Foo037(t *testing.T) {
        	tests := []struct {
        		name string
        		smtg *someIndirectImportedStruct
        	}{
        		// TODO: Add test cases.
        	}
        	for _, tt := range tests {
        		smtg := &someIndirectImportedStruct{}
        		smtg.Foo037()
        	}
        }
    gotests_test.go:805: /tmp/gotests_test343074369/receiver_is_indirect_imported_struct.go
FAIL
FAIL	github.com/cweill/gotests	4.784s
ok  	github.com/cweill/gotests/gotests	(cached)
ok  	github.com/cweill/gotests/gotests/process	0.006s
?   	github.com/cweill/gotests/internal/goparser	[no test files]
?   	github.com/cweill/gotests/internal/input	[no test files]
?   	github.com/cweill/gotests/internal/models	[no test files]
?   	github.com/cweill/gotests/internal/output	[no test files]
?   	github.com/cweill/gotests/internal/render	[no test files]
?   	github.com/cweill/gotests/internal/render/bindata	[no test files]
?   	github.com/cweill/gotests/templates	[no test files]
FAIL
soren@potassium:~/src/gotests$ git rev-parse HEAD
2435ae532b971c836a98aa497a0f3a07b73c3488
soren@potassium:~/src/gotests$ git status
On branch develop
Your branch is up to date with 'origin/develop'.

nothing to commit, working tree clean
soren@potassium:~/src/gotests$ 
```

**Comment by sorenh** [comment]
To be honest, I feel like the `got` is more correct than the `want`. Why provide a `*someIndirectImportedStruct` in the test table, but then not use it in the body of the test?

**Comment by cweill** [comment]
This should be fixed with #150.
